streams/src/test/java/org/apache/kafka/streams/processor/internals/SinkNodeTest.java

```diff
@@ -51,6 +51,8 @@ public void shouldThrowStreamsExceptionOnInputRecordWithInvalidTimestamp() {
             sink.process(anyKey, anyValue);
             fail("Should have thrown StreamsException");
         } catch (final StreamsException ignored) {
+        } finally {
+            context.close();
```

Comment:
Could we move `context` as this class's member variable, and then close the context in `@After`? All test cases seem to create the same object.